### PR TITLE
*: tiny cleanup

### PR DESCRIPTION
**Pull request by BusyJay**
- update rust-rocksdb to latest version
- cleanup some codes
- make `assert_quorum` more informative

After this pr, it should be able to compile TiKV on Windows, not a big deal, but still a nice-to-have. :)

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @hhkbp2

File: tests/raftstore/cluster.rs [hunk 1]
```diff
@@ -370,11 +370,27 @@ impl<T: Simulator> Cluster<T> {
...
-            return true;
+            return;
         }
-        self.engines.values().filter(condition).count() > self.engines.len() / 2
+        let half = self.engines.len() / 2;
+        let mut qualified_cnt = 0;
+        for (id, engine) in &self.engines {
+            if !condition(engine) {
+                debug!("store {} is not qualified yet.", id);
+                return;
```
Comment (hhkbp2) [inline tests/raftstore/cluster.rs]: Why return earlier without checking the others?

**Comment by hhkbp2** [comment]
Rest LGTM

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**